openmetadata-service/src/main/java/org/openmetadata/service/resources/types/TypeResource.java

```diff
@@ -521,6 +525,89 @@ public Response getAllCustomPropertiesByEntityType(
     }
   }
 
+  @GET
+  @Path("/customProperties/search")
```

Comment:
lets not move this to another end point, all of this should be done in search resouce

Reply:
yes we can move it to search resource